redis-lock/src/main/java/com/netflix/conductor/locking/redis/RedisLock.java

```diff
@@ -0,0 +1,129 @@
+package com.netflix.conductor.locking.redis;
+
+import com.google.inject.Inject;
+import com.google.inject.ProvisionException;
+import com.netflix.conductor.core.config.Configuration;
+import com.netflix.conductor.core.utils.Lock;
+import com.netflix.conductor.locking.redis.config.RedisLockConfiguration;
+import com.netflix.conductor.metrics.Monitors;
+import org.apache.commons.lang3.StringUtils;
+import org.redisson.Redisson;
+import org.redisson.api.RLock;
+import org.redisson.api.RedissonClient;
+import org.redisson.config.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+
+public class RedisLock implements Lock {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RedisLock.class);
+
+    private RedisLockConfiguration configuration;
+    private Config redisConfig;
+    private RedissonClient redisson;
+    private final int connectionTimeout = 10000;
+    private static String LOCK_NAMESPACE = "";
+
+    @Inject
+    public RedisLock(RedisLockConfiguration configuration) {
+        this.configuration = configuration;
+        LOCK_NAMESPACE = configuration.getProperty("decider.locking.namespace", "");
+        RedisLockConfiguration.REDIS_SERVER_TYPE redisServerType = configuration.getRedisServerType();
```

Comment:
This statement is repeated outside and within the `try{}` block, is this intentional?